### DISCUSSION TITLE
More than one target

### DISCUSSION THREAD

**Discussion by alejandrozf** [bigskysoftware/htmx#1624] [Q&A] [answered]
Hi, i'm new to HTMX

Wanna know how one can modify more than one target at the same time

Thanks in advance

**Comment by Telroshan** [comment, marked as answer]
Hello, you have several available possibilities, it's up to you to pick the one that suits your situation the best:
- [hx-swap-oob](https://htmx.org/attributes/hx-swap-oob/)
- [hx-select-oob](https://htmx.org/attributes/hx-select-oob/)
- The [multi-swap](https://htmx.org/extensions/multi-swap/) extension

Check the linked docs of each to see their syntax & usage
Hope this helps!

**Comment by alejandrozf** [comment]
Awesome! thank you! That's what i needed!
